test/numerical/RandomNumberGenerator.jl

```diff
@@ -0,0 +1,2 @@
+CRRao.set_rng(StableRNG(123))
+@test rand(CRRao.CRRao_rng, Float64, 1) ≈ [0.18102554215580358]
```

Comment:
@ayushpatnaikgit 
I'm performing an approximate equality check here. Do you want exact equality instead?

Reply:
approximate equality check should be fine